config/locales/server.en.yml

```diff
@@ -1719,7 +1719,7 @@ en:
     blocked_crawler_user_agents: "Unique case insensitive word in the user agent string identifying web crawlers that should not be allowed to access the site. Does not apply if allowlist is defined."
     slow_down_crawler_user_agents: 'User agents of web crawlers that should be rate limited as configured in the "slow down crawler rate" setting. Each value must be at least 3 characters long.'
     slow_down_crawler_rate: "If slow_down_crawler_user_agents is specified this rate will apply to all the crawlers (number of seconds delay between requests)"
-    content_security_policy: "Enable Content-Security-Policy (CSP)"
+    content_security_policy: "Activates the Content-Security-Policy (CSP), an additional layer of security that helps to prevent certain types of attacks, including Cross Site Scripting (XSS) and data injection. When enabled, the server provides the security policy in the form of a 'Content-Security-Policy' HTTP header."
```

Comment:
I would keep "Enable Content-Security-Policy (CSP)`, we usually don't use the "Activate" wording for enabling site settings. Also the phrase "When enabled, the server provides the security policy in the form of a 'Content-Security-Policy' HTTP header" is redundant, admins shouldn't care how the CSP is delivered. (And I'm pretty sure it can _only_ be a header.) 

```suggestion
    content_security_policy: "Enable Content-Security-Policy (CSP). CSP is an additional layer of security that helps to prevent certain types of attacks, including Cross Site Scripting (XSS) and data injection."
```

Reply:
Sounds good :+1: